lms/djangoapps/static_template_view/views.py

```diff
@@ -92,12 +94,32 @@ def render_press_release(request, slug):
         return resp
 
 
+@fix_crum_request
```

Comment:
Is this decorator used mostly for static error pages? I'm trying to figure out when this should be used/how I would figure out I need this on a view.

Reply:
Yea, that was my reading of the docs on it.  When an error occurs, the `crum` package can't properly fetch the request object and so decorating the package like this ensures that the correct request is fetch able.